.env.development

```diff
@@ -2,7 +2,8 @@ VITE_TRACTION_BASE_URL=http://traction
 VITE_TRACTION_GRAPHQL_URL=http://traction/graphql
 VITE_PRINTMYBARCODE_BASE_URL=http://printmybarcode
 VITE_SAMPLEEXTRACTION_BASE_URL=http://sampleextraction
-VITE_LABWHERE_BASE_URL=http://labwhere
+VITE_LABWHERE_BASE_URL=http://localhost:3003
 VITE_SEQUENCESCAPE_API_KEY=development
 VITE_LOG=true
-VITE_ENVIRONMENT=development
\ No newline at end of file
+VITE_ENVIRONMENT=development
+VITE_DESTROYED_LOCATION_BARCODE=lw-destroyed-217
```

Comment:
Are youn planning to add the destroyed location to the deployment project or just the .env files?

Reply:
It is already added in deployment project https://github.com/sanger/deployment/pull/533